denizen_scripts/global/server/network_item.dsc

```diff
@@ -62,6 +62,7 @@ network_item_events:
 network_item_inventory:
   type: inventory
   inventory: chest
+  title: <&a>Play Menu
   definitions:
     filler: <item[white_stained_glass_pane]>
```

Comment:
future reference, this doesn't need an item wrapper if we can fix this the next time we touch this file